data/models/4x-MinecraftAlphaUpscaler-with-Good-data.json

```diff
@@ -2,7 +2,10 @@
     "name": "MinecraftAlphaUpscaler with Good data",
     "author": "washed-up",
     "license": "GPL-3.0-only",
-    "tags": [],
+    "tags": [
+        "game-textures",
+        "restoration"
+    ],
```

Comment:
This model, as well as `4x-Minepack`, are not for Minecraft textures, but one specific screenshot of a minecraft world specifically: pack.png. This image is a thumbnail is the game, and the community had a huge search for the seed of the world that generated this image some years back. These models were used to upscale pack.png since the image is only 128x128px.

Here's a video about this whole ordeal: https://www.youtube.com/watch?v=ea6py9q46QU

Reply:
Hmm. I was aware of this, but I wasn't sure what to list it under. Technically it was a file from a game 😛 

What do you feel it would fit better under?

Reply:
Hmmm, not sure. Do we have something for video game screenshots?

Reply:
> Hmmm, not sure. Do we have something for video game screenshots?

As far as I know, we don't. It might be good though, as there's a few models dedicated to that purpose

Reply:
@RunDevelopment I added the Game Screenshots tag. Hopefully this resolves the issue